### PR TITLE
Adds chrome 2.0 support🤪🤷‍♀️⏱

### PR DESCRIPTION
**Pull request by AllenBW**
n rename url from /content-preview to /preview

### CODE REVIEW

File: src/SmartComponents/Recs/_Details.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import '~@redhat-cloud-services/frontend-components-inventory-insights/esm/Insights.css';
+@import '~@redhat-cloud-services/frontend-components-inventory-insights/Insights/Insights.scss';
```
Comment (mhuth) [inline src/SmartComponents/Recs/_Details.scss:1]: This failed with `Insights.scss` for me and the file on my systems was `insights.scss`

**Review by mhuth — APPROVED** [review]
Couldn't properly get it to display due to `'(failed)net::ERR_CERT_INVALID'` but I assume it works file (well, except for that `insights.scss` file issue I mentioned).  You'll have to show me how to resolve the invalid cert thing.

Update: nevermind, I got it working (by browsing to https://10.72.32.104:8090/hits/ and accepting the unsafe connection).  Works well :+1:

File: src/SmartComponents/Recs/_Details.scss (hunk 1 above)
Comment (AllenBW) [reply src/SmartComponents/Recs/_Details.scss:1]: did yah blow away n reinstall  yer node modules? they redid those packages... where things are (in them) changed
🤔

Comment (mhuth) [reply src/SmartComponents/Recs/_Details.scss:1]: > did yah blow away n reinstall yer node modules? they redid those packages... where things are (in them) changed

I didn't.  Just tried it and it worked :facepalm:

**Pull request merged**